node/node.go

```diff
@@ -205,7 +206,16 @@ func New(params NodeParams) (*Node, error) {
 		if !params.CreateWallet {
 			return nil, nil
 		}
-		return wallet.New(cs, tp, filepath.Join(dir, modules.WalletDir))
+		w, err := wallet.New(cs, tp, filepath.Join(dir, modules.WalletDir))
```

Comment:
And same with this stuff, creation of a key and initialization of the wallet should happen in the `siatest` package when you create a server/client pair, not in the node package.